DOCKERHUB.md

```diff
@@ -0,0 +1,83 @@
+# Kroki
+
+Convert plain text diagrams to images !
+
+Kroki provides a unified API with support for BlockDiag (BlockDiag, SeqDiag, ActDiag, NwDiag, PacketDiag, RackDiag), BPMN, Bytefield, C4 (with PlantUML), Ditaa, Erd, Excalidraw, GraphViz, Mermaid, Nomnoml, Pikchr, PlantUML, SvgBob, UMLet, Vega, Vega-Lite, WaveDrom... and more to come!
+
+## Getting Started
+
+### Prerequisities
+
+In order to run this container you'll need docker installed.
+
+* [Windows](https://docs.docker.com/docker-for-windows/)
+* [OS X](https://docs.docker.com/docker-for-mac/)
+* [Linux](https://docs.docker.com/get-started/)
+
+### Usage
+
+See also the [installation](https://docs.kroki.io/kroki/setup/configuration/) docs in the [Kroki documentation](https://docs.kroki.io/)
+
+#### start a kroki instance
+
+    $ docker run --name some-kroki -d yuzutech/kroki
+
+This image includes `EXPOSE 8000` (the kroki port), so standard container linking will make it automatically available to the linked containers. If you want to map port 8000 in the container to a port on your host, please use the `--publish` or `-p` flag:
+
+    $ docker run -p8000:8000 --name some-kroki -d yuzutech/kroki
+
+#### connect with companion containers
+
+We recommend using docker-composer to connect with companion containers, such as 
+[kroki-bpmn](https://hub.docker.com/r/yuzutech/kroki-bpmn), [kroki-blockdiag](https://hub.docker.com/r/yuzutech/kroki-blockdiag), [kroki-excalidraw](https://hub.docker.com/r/yuzutech/kroki-excalidraw) and/or [kroki-mermaid](https://hub.docker.com/r/yuzutech/kroki-mermaid)
+```
+version: "3"
+services:
+  core:
+    image: yuzutech/kroki
+    environment:
+      - KROKI_BLOCKDIAG_HOST=blockdiag
+      - KROKI_MERMAID_HOST=mermaid
+    ports:
+      - "8000:8000"
+  blockdiag:
+    image: yuzutech/kroki-blockdiag
+    ports:
+      - "8001:8001"
+  mermaid:
+    image: yuzutech/kroki-mermaid
+    ports:
+      - "8002:8002"
+  bpmn:
+    image: yuzutech/kroki-bpmn
+    ports:
+      - "8003:8003"
+  excalidraw:
+    image: yuzutech/kroki-excalidraw
+    ports:
+      - "8004:8004"
+```
+
+If you don't want to use `docker-compose`, you can configure the host and port for each companion container using environment variables:
+
+|Container         | Host                  | Port                  |
```

Comment:
I did format the table better. 

(I don't think docker hub description could be written in asciidoc unfortunately)